frontend/include/chpl/resolution/resolution-types.h

```diff
@@ -1119,7 +1270,16 @@ class MostSpecificCandidate {
   }
 
   bool operator==(const MostSpecificCandidate& other) const {
+    bool faMapsEqual = faMap_ == other.faMap_;
+    if (!faMapsEqual && faMap_ && other.faMap_) {
+      // Try a deep comparison to avoid unnecessary cache invalidation when
+      // bumping generations. The pointers may differ, but contain the enclosed
```

Comment:
contain -> contents of?

Reply:
I think `contain` is a remnant of a different sentence I edited away. I reworded the comment.